routes/api.php

```diff
@@ -14,7 +16,7 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
+Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
```

Comment:
Do not use anonymous functions in router. They cannot be cached, move it to some controller.